### PR TITLE
Update setup.py

### PR DESCRIPTION
**Pull request by MauriceHendrix**
## Description

I updated setup.py

## Motivation and Context

test Pypi  didn't like the way it was trying to read the license and it repented the license to the description and quoted the (markdown) description, not a good look. This version shows up better

### CODE REVIEW

File: setup.py [hunk 1]
```diff
@@ -4,18 +4,21 @@
...
 
-with open('LICENSE') as f:
-    license_ = f.read()
-
 setup(
     name='cellmlmanip',
     version='0.1.0',
     description='CellML loading and model equation manipulation',
     long_description=readme,
+    long_description_content_type="text/markdown",
```
Comment (MichaelClerx) [inline setup.py:12]: We're using `'` in the rest of the arguments

Comment (MauriceHendrix) [reply setup.py]: sure

File: setup.py [hunk 2]
```diff
@@ -4,18 +4,21 @@
...
     version='0.1.0',
     description='CellML loading and model equation manipulation',
     long_description=readme,
+    long_description_content_type='text/markdown',
     author='Asif U Tamuri, Sarah M Keating, Maurice Hendrix, Michael Clerx, Jonathan Cooper',
     author_email='j.p.cooper@ucl.ac.uk',
     url='https://github.com/ModellingWebLab/cellmlmanip',
-    license=license_,
+    classifiers=[
+        "Programming Language :: Python :: 3",
```
Comment (MichaelClerx) [inline setup.py:17]: And here

Comment (MauriceHendrix) [reply setup.py]: yes but are you ok with the change in principle?

**Pull request merged**